packages/vanchor-client/src/queries/TotalValueLocked.ts

```diff
@@ -0,0 +1,106 @@
+import { execute } from "../../.graphclient"
+
+export interface TotalValueLockedByChain { chainName: string, totalValueLocked: number }
+
+export interface TotalValueLockedByChainAndByToken extends TotalValueLockedByChain { tokenSymbol: string }
+
+export interface TotalValueLockedByVAnchor { vAnchorAddress: string, totalValueLocked: number }
+
+export const GetVAnchorTotalValueLockedByChain = async (chainName: string, vanchorAddress: string): Promise<TotalValueLockedByChain> => {
+  const query = `
+  query TotalValueLocked {
+  vanchorTotalValueLocked(id: "${vanchorAddress.toLowerCase()}"){
+    totalValueLocked
+  }
+}
+`
+  const result = await execute(query, {}, {
+    chainName,
+  })
+
+  return {
+    totalValueLocked: result.data.vanchorTotalValueLocked?.totalValueLocked,
+    chainName: chainName
+  }
+}
+
+export const GetVAnchorTotalValueLockedByChains = async (chainNames: Array<string>, vanchorAddress: string): Promise<Array<TotalValueLockedByChain>> => {
+  const promises: Array<Promise<TotalValueLockedByChain>> = [];
+
+  for (const chainName of chainNames) {
+    promises.push(GetVAnchorTotalValueLockedByChain(chainName, vanchorAddress))
+  }
+
+  return await Promise.all(promises);
+}
+
+export const GetVAnchorsTotalValueLockedByChain = async (chainName: string, vanchorAddresses: Array<string>): Promise<Array<TotalValueLockedByVAnchor>> => {
+  const query = `
+  query TotalValueLockedByVAnchor {
+  vanchorTotalValueLockeds(
+    where: {id_in: [${vanchorAddresses.map((address) => '\"' + address.toLowerCase() + '\"').join(",")}]}
+  ) {
+    id
+    totalValueLocked
+  }
+}
+`
```

Comment:
is that https://github.com/Thinkmill/ts-gql something we could utilize to help with this?